api/auth/manageCasesUserRoleAuth.ts

```diff
@@ -0,0 +1,15 @@
+export const CASEWORKER_INDENTIFIER = 'caseworker'
```

Comment:
In 12FA, maybe we should extract this into an environment variable?

Reply:
We need abstract static data out in env var into our config

Reply:
The 'casworker' role does not change with environments, why extracting it to an   environment variable ?

Reply:
I don't mind changing it, but we currently have probably around +400 users in the wild with 'caseworker-' roles. I would argue that it might be better to keep it here; as it's unit testable therefore we can make sure it doesn't change from 'caseworker'. If we have it as a configuration parameter, and someone accidentally changes it, it would mean that no one login, therefore I believe that having it as a config parameter increases our risk. 

But then if the configuration parameters are locked down on Production it should be fine.

Reply:
'caseworker' is now coming from the configuration parameters.